backend/climateconnect_api/factories.py

```diff
@@ -0,0 +1,12 @@
+import factory
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', 'password', 'is_active')
+
+    is_active = True
+    username="test_user" 
```

Comment:
Is this a fallback username?

Reply:
No, it was for testing only. Removed it. Thanks!